CHANGELOG.rst

```diff
@@ -13,6 +13,14 @@ its best to adhere to `Semantic Versioning <https://semver.org/spec/v2.0.0.html>
 List entries are sorted in descending chronological order. Contributors to each release
 were listed in alphabetical order by first name until version 0.7.0.
 
+0.8.3 (2023-03-22)
+==================
+
+Changed
+-------
+- Inserted convention "Bruker" into print statement produced by ``EBSD`` method ``hough_indexing()`` to avoid confusion around displayed projection center. 
```

Comment:
Would shorten the description, to something like "EBSD.hough_indexing() info message now informs that the given projection center is in Bruker's convention".

Also, could you keep the line length to the same as the remaining changelog? Makes it easier to read in editors that don't automatically wrap lines.

Reply:
Yes that is a better description, changing it right away!